docs/how-tos/howto-run-loadbot.md

```diff
@@ -0,0 +1,80 @@
+---
+id: howto-run-loadbot 
+title: How to run the loadbot for stress testing your network
+---
+
+## Prerequisites
+
+This how-to assumes you have followed
+the [guide on how to set up an IBFT cluster locally](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally)
+or [guide on how to set up an IBFT cluster on the cloud](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud).
+The loadbot also works when using dev mode.
+
+A functioning node is required to stress test the network.
+
+When building a network using the Polygon SDK, you may want to test how many transactions your configuration can handle
+per second, or how your network will react when you submit valid and invalid transactions.
+
+The Polygon SDK binary includes a `loadbot` command, responsible for sending a specific number of transactions per
+second.
+
+This _how-to_ will explain how to run the loadbot on a single node configuration using 2 accounts prefilled with a
+balance of 1000 ETH each.
+
+:::note
+
+This is not the only parameter you can specify when running the loadbot.
```

Comment:
We can add a link to the actual loadbot command / flags here, in the Commands section

Reply:
Speaking of which, I don't see the command added to the CLI Commands section.

We should update that document as well 👍